script.js

```diff
@@ -1,12 +1,27 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let colorlist = ['brown', 'red']
+let stroke_color = ['brown, red']
+let backImg; 
+let length = 10;
+
 
 function setup() {
   createCanvas(windowWidth, windowHeight);
-    background(255);
+  //background(255);
+ 
 }
 
 function draw() {
-  noStroke();
+  if (mouseIsPressed){
+    var audio = new Audio("https://interactive-examples.mdn.mozilla.net/media/cc0-audio/t-rex-roar.mp3");
+    audio.play();
```

Comment:
but watch out! this plays the same audio files many many times every frame that the mouse is held down. that's why it sounds a little choppy. you want to update your code so that it only plays once, when the mouse is pressed initially.